packages/@aws-cdk/aws-cloudformation/lib/pipeline-actions.ts

```diff
@@ -48,6 +48,7 @@ export abstract class PipelineCloudFormationAction extends codepipeline.DeployAc
     constructor(parent: cdk.Construct, id: string, props: PipelineCloudFormationActionProps, configuration?: any) {
```

Comment:
Construct initializers should never have 4 parameters. Why do we need `configuration` externally to `props`?

Reply:
This is just an abstract class, so I don't think it really matters. Also, this is the way it's modeled now, I haven't touched it in this PR.